ariadne_codegen/contrib/shorter_results.py

```diff
@@ -99,6 +104,19 @@ def generate_result_class(
             class_def, operation_definition, selection_set
         )
 
+    def generate_fragments_module(
+        self,
+        module: ast.Module,
+        fragments_definitions: Dict[str, FragmentDefinitionNode],
+    ) -> ast.Module:
+        """Store a map of all fragment classes and their AST."""
+        for fragment_class in [
+            x.name for x in module.body if isinstance(x, ast.ClassDef)
+        ]:
+            self.imported_types[fragment_class] = ".fragments"
```

Comment:
`"fragments"` is only a default value, we shouldn't hardcode it. A dictionary with settings is passed to every `Plugin`, so you can get name like this:
```py
fragments_module_name = self.config_dict.get("tool", {}).get("ariadne-codegen", {}).get("fragments_module_name", "fragments")
```

Reply:
How should I approach the relative import path? When looking at this in test and my example I don't have a `fragments_module_name`. If that is present, would it be prefixed with a dot? And should I default to `.fragments` instead?

Reply:
It's only a filename of a file that we generate, similar to `enums_module_name`, so it's safe to assume it will always be in generated package's directory. Either you can prefix it with `.` or change the [logic](https://github.com/mirumee/ariadne-codegen/blob/main/ariadne_codegen/contrib/shorter_results.py#L155) to set the [level](https://github.com/mirumee/ariadne-codegen/blob/main/ariadne_codegen/codegen.py#L28) of generated import to `1` when importing a fragment.

Reply:
Cool, thanks! I opted to change the logic since we already have the dot/level set for the regular imports. Added a custom name for the test to ensure it gets picked up properly as well.